typescript/sdk/src/deploy/verify/utils.ts

```diff
@@ -1,11 +1,15 @@
 import { ethers, utils } from 'ethers';
 
+import { ZKSyncArtifact } from '@hyperlane-xyz/core';
 import { Address, eqAddress } from '@hyperlane-xyz/utils';
 
+import { ExplorerFamily } from '../../metadata/chainMetadataTypes.js';
 import { ChainMap, ChainName } from '../../types.js';
 
 import { ContractVerificationInput } from './types.js';
 
+const { Interface } = await import('@ethersproject/abi');
```

Comment:
Why are you doing a dynamic import of this? Instead, import it directly from the `ethers` package like other imports above

Reply:
Resolved